### ISSUE TITLE
Feed fails to update in fresh FF profile, feed worked in profile created pre-WebExtension

### ISSUE THREAD

**Issue by jtdor** [brief-rss/brief#365]
Hi there, one of the feeds I use (https://news.ycombinator.com/rss) fails to update after I created a fresh FF profile and subscribed to it. The feed worked for me til a few days ago with an old FF profile. In the old profile (and pre-WebExtension) I had a folder of Live Bookmarks where I had Brief look for feeds.

I took a look into the debugger when updating this feed:
```
Brief: enqueued 1 feeds  updater.js:68:9
Brief: fetch error DOMException [DataError: "Data provided to an operation does not meet requirements."
code: 0
nsresult: 0x80660005
location: moz-extension://4af37e3c-b7a4-4358-90f3-c0560174fb6d/scripts/database.js:1440]  updater.js:136:31
	_worker/< moz-extension://4af37e3c-b7a4-4358-90f3-c0560174fb6d/scripts/updater.js:136:31
Brief: update finished  updater.js:187:9
```

Unsubscribing and the re-subscribing again gives me this:
```
Error: No matching message handler  undefined
Restoring hidden feed Object { feedID: "c90003bdb1e63545fa593d2128d658a7420…", feedURL: "https://news.ycombinator.com/rss", title: "", rowIndex: 30, isFolder: false, parent: "-1", hidden: 1528059405691, entryAgeLimit: 0, maxEntries: 0, updateInterval: 0, 11 more… }  database.js:382:17
[Exception... "Component returned failure code: 0xc1f30001 (NS_ERROR_NOT_INITIALIZED) [nsIMessageSender.sendAsyncMessage]"  nsresult: "0xc1f30001 (NS_ERROR_NOT_INITIALIZED)"  location: "JS frame :: resource://gre/modules/ExtensionUtils.jsm :: sendAsyncMessage :: line 533"  data: no]  (unknown)
	sendAsyncMessage resource://gre/modules/ExtensionUtils.jsm:533:51
	_handleMessage/deferred.promise< resource://gre/modules/MessageChannel.jsm:984:9
Brief: enqueued 1 feeds  updater.js:68:9
```

Happy to help with any further debugging. Observed with Brief 2.5.6 and FF 60.

**Comment by tanriol** [comment]
Thank you, a very nice bug report. Fix ready for the next version.

**Comment by jtdor** [comment]
Ah, that's a quick fix.

Well, thank you for the useful extension. (I hope there will be any practical way to bring back syncing of the feed list to make it even more useful. Even if it’s just creating a bookmark for each feed URL and let that sync through FF sync, in lack of other APIs.)

**Comment by tanriol** [comment]
Should be fixed in 2.5.7.

> I hope there will be any practical way to bring back syncing of the feed list to make it even more useful.

There are APIs and there are some ideas, but testing them requires more time than I have.

**Comment by tanriol** [comment]
Should be fixed in 2.5.7.
